src/Persistence_SQL.php

```diff
@@ -590,7 +590,11 @@ public function action($m, $type, $args = [])
      */
     public function tryLoad(Model $m, $id)
     {
-        $load = $this->action($m, 'select');
+        if (!$m->id_field) {
```

Comment:
Shouldn't this be in Model->tryLoad or Persistence->tryLoad instead of Persistence_SQL?
Is there any use case when model can be loaded without knowing its id?